stories/Containers/StoreHoursList.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { StoreHoursList } from '../../src';
+
+storiesOf('StoreHoursList', module)
+    .add('with default', () => (
+        <StoreHoursList/>
```

Comment:
How will this component remember which category was selected as active? How will this component know which data it contains?

The UI looks great, it's on the right track, the code structure has some design issues that can easily be fixed as you start sorting your code in the correct place :)